.eslintrc.js

```diff
@@ -2,12 +2,17 @@ module.exports = {
   "env": {
     "browser": true,
     "es2021": true,
-    "node": true
+    "node": true,
+
+  },
+  "globals": {
+    "gruntConfig": "readonly"
```

Comment:
What behaviour does this affect with Grunt?

Reply:
It fixes this warning/error:
![Screenshot from 2021-12-06 15-43-43](https://user-images.githubusercontent.com/77306764/144940280-eda56da4-c9da-488f-9b42-753e97e7709e.png)

Reply:
https://eslint.org/docs/rules/no-undef

Reply:
Readonly is an eslint thing, not a grunt thing, so it shouldn't have any effect on grunt. When a variable is set to readonly, eslint will [give you a warning if you try to change it](https://smartdevpreneur.com/how-and-when-to-specify-globals-in-eslintrc-a-detailed-example-with-configs-and-results/#:~:text=about%20importing%20React.-,Readonly,-and%20Writable%20Globals) (Error: Read-only global 'var1' should not be modified. eslint (no-global-assign)